putio.py

```diff
@@ -255,6 +280,11 @@ def add_torrent(cls, path, parent_id=0, extract=False, callback_url=None):
     def clean(cls):
         return cls.client.request('/transfers/clean', method='POST')
 
+    @classmethod
+    def cancel(cls, transfer_ids):
+        return cls.client.request('/transfers/cancel', method='POST',
+                                   data={'transfer_ids': ','.join(transfer_ids)})
+
```

Comment:
Every other method on that class is a classmethod. There is no instance. It makes sense to stay a classmethod because it mirrors the action taken in the UI, which is to act on the collection, by providing a list of transfers to cancel.

If the API had an /transfers/<transfer_id>/cancel endpoint, then an instance method would make sense.